webpack.config.js

```diff
@@ -48,49 +48,57 @@ const extractConfig = {
 							secondary: '#11a0d2',
 							toggle: '#11a0d2',
 							button: '#0085ba',
+							outlines: '#007cba',
 						},
 						themes: {
```

Comment:
Unrelated to this PR, but it seems weird we put themes in the webpack config. Any context for why we do that or should I make an issue to move 'em out?

Reply:
We put theme here to automatically generate the required CSS files using a post-css plugin #6739

Reply:
We can already use `postcss.config.js` and move all the config their. I personally don't like the proliferation of root level config files, but no strong opinion.